articles/quickstart/native/_includes/_facebook_login_howto.md

```diff
@@ -0,0 +1,60 @@
+## Setup the “Continue with Facebook” Button
+
+### Requesting Facebook permissions
```

Comment:
```suggestion

This guide will help you add authentication with Auth0 to the application you built in the first step.

### Requesting Facebook permissions
```

Reply:
I think it's not 100% clear that the suggested changes are going to be applied over the app that was created in the 'Before you start' section.